frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -44,33 +47,41 @@ const Details_ = ({flags, obj: pvc}) => {
   const storageClassName = _.get(pvc, 'spec.storageClassName');
   const volumeName = _.get(pvc, 'spec.volumeName');
   const requestedStorage = _.get(pvc, 'spec.resources.requests.storage');
-  const storage = _.get(pvc, 'status.capacity.storage');
+  const volumeMode = _.get(pvc, 'status.volumeMode');
   const accessModes = _.get(pvc, 'status.accessModes');
   return <div className="co-m-pane__body">
     <SectionHeading text="PersistentVolumeClaim Overview" />
     <div className="row">
-      <div className="col-sm-6">
-        <ResourceSummary resource={pvc}>
+      <div className="col-sm-4">
+        <ResourceSummary resource={pvc} showPodSelector={false} showNodeSelector={false}>
```

Comment:
This change is unnecessary and should be rolled back.

```suggestion
        <ResourceSummary resource={pvc}>
```